simulation/utils/simulation_pygame_utils.py

```diff
@@ -1,7 +1,12 @@
 import pygame
+import pymunk
+
 from pygame.color import THECOLORS
 
-from utils.simulation_pymunk_utils import SCREEN_HEIGHT
+from utils.simulation_pymunk_utils import SCREEN_HEIGHT, Color
+
+GOAL_OBJECT_COLOR = Color(51, 153, 255)
```

Comment:
Wrzuciłem to już na PR @PatrykRadon . Zmienić te Color() na jakieś słowne wartości w jakimś enumie wcześniej czy coś, żeby było widać na pierwszy rzut oka jaki kolor ma mapa a jaki cel

Reply:
To będzie koszmar z merge'owaniem tego wszytkiego XD